lib/network.js

```diff
@@ -6,14 +6,6 @@ module.exports = function (archive, opts) {
   opts = opts || {}
 
   var swarm = hyperdiscovery(archive, opts)
-  var network = {
-    swarm: swarm,
-    options: swarm.options || opts
-  }
-
-  network.__defineGetter__('connected', function () {
-    return swarm.connected
-  })
-
-  return network
+  swarm.options = swarm.options || opts
```

Comment:
Is this line necessary? Confused about if `opts` above becomes `swarm.options` under the hood